AutoGVP/01-annotate_variants_CAVATICA_input.R

```diff
@@ -56,11 +56,11 @@ option_list <- list(
               help = "variant_summary file (format: variant_summary_2023-02.txt)"),
   make_option(c("--submission_summary"), type = "character",
               help = "specific submission summary file (format: submission_summary.txt)"),
-  make_option(c("--gnomad_variable"), type = "character",default = "Freq_gnomAD_genome_ALL",
+  make_option(c("--gnomad_variable"), type = "character",
```

Comment:
can you update the name of this variable to `pop_variable` or `popfreq_variable` for population frequency? This can be gnomad, exac, or 1000g, or even a custom population, so we should not make it `gnomad`